assets/app/styles/_tables.less

```diff
@@ -61,6 +61,26 @@
   }
 }
 
+.data-toolbar {
```

Comment:
Suggest `.data-toolbar, .table-toolbar` here instead of repeating the styles for vertical-divider.